.pre-commit-config.yaml

```diff
@@ -23,3 +23,8 @@ repos:
         entry: bash -lc 'pnpm --filter @promethean/image-link-generator run build && pnpm tsx scripts/generate-layer-images.ts'
         language: system
         pass_filenames: false
+      - id: eslint-diff
```

Comment:
this gives us lint feed back on just what has changed, so we at least do not introduce new problems.